src/main/java/com/fvv/bookstore/view/OrderViewImpl.java

```diff
@@ -94,6 +95,109 @@ public void addOrder() {
 		}
 	}
 
+	/**
+	 * {@inheritDoc}
+	 */
+	@Override
+	public void listTotalOrdersMonth() {
+		try {
+			StringBuilder sb = new StringBuilder();
+			Integer monthSearch = Integer.parseInt(JOptionPane.showInputDialog("Insert the number month to search"));
+			Integer yearSearch = Integer.parseInt(JOptionPane.showInputDialog("Insert the year to search"));
+			List<Order> orders = this.orderController.listTotalOrdersMonth(monthSearch, yearSearch);
+			if(orders != null && !orders.isEmpty()) {
+				Integer quantity = this.orderController.calculateQtyOrders(orders);
+				Double amount = this.orderController.calculateTotalOrders(orders);
+				sb.append("Quantity of orders: ").append(quantity).append(" - Amount in the month: R$").append(amount).append(Constants.LINE_SEPARATOR)
+				.append(Constants.LINE_SEPARATOR);
+				for(Order ord : orders) {
+					sb.append("Date: ").append(ord.getCreationDate()).append(" - Order ID: ").append(ord.getId()).
```

Comment:
Format the date in a better format for example 22/05/2018 (as mentioned before, try to use the same format throughout the all inputs and outputs

Reply:
ok